marsha/core/templates/core/lti_video.html

```diff
@@ -0,0 +1,7 @@
+<html>
+  <head>
+  </head>
+  <body>
+    <div data-state="{{ state }}" {% for key, value in policy.items %} data-{{ key }}="{{ value }}"{% endfor %}></div>
```

Comment:
not sure the state/role should be mixed in with the policy keys?

Reply:
Yes that's up to you. The way that makes sense and is easy for you to catch ?

Reply:
I'll update it later on then. 🚢 it

Reply:
Yeah that's what I thought.